firmware/gen_config.sh

```diff
@@ -55,18 +49,4 @@ for BOARD in \
  [ $? -eq 0 ] || { echo "ERROR generating board $BOARD_NAME $BOARD_SHORT_NAME $INI"; exit 1; }
 done
 
-#
-# TODO: it's time to kill the 'default' bundle concept and just live happily with explicit f407-discovery
-#
-# problem statement: it's desired that plain 'make' invocation compiles discovery binary
-# reality: while we have per-target signature*.h and rusefi*.ini we do not have matching per-target 'engine_configuration_generated_structures.h'
-# so for plain 'make' to produce f4 discovery we rely of file system engine_configuration_generated_structures.h being f4 discovery version
-#
-#
-# default config should be generated after normal custom boards so that it would be default
-# firmware/controllers/generated/rusefi_generated.h file which would be pushed into VCS
-./gen_config_default.sh
```

Comment:
I can't find an existing flow that relies on this invocation here; `gen-configs.yaml` already has this explicitly after the call to this `gen_config.sh` and is the only flow I could find, unless somehow the GH build containers are re-used on purpose in some scenario/way I'm unaware of.